tests/functional/Parser/Fixtures/Assignments/condition_elseif.typoscript

```diff
@@ -1,5 +1,6 @@
 [globalVar = GP:foo=1]
 foo = bar
+[global]
```

Comment:
Is this possibly a leftover from another change? Like #52 or something?

Reply:
No, i wanted to test if the closing [global] is added. I know, that the old behaviour is not tested anymore. I am gonna revert it and only test the old behaviour because it is a way more complicated and should be tested.